app/components/edit/ProvidedService.jsx

```diff
@@ -192,6 +192,7 @@ Service
           </li>
 
           <EditSchedule
+            canInheritFromParent
```

Comment:
Just wondering, when would we not want to set this flag to true?

Reply:
Ya, that's a good question. We reuse `EditSchedule` for organizations (resources?) and for services. We only want services to be able to inherit schedules since organizations don't have a 'parent' to inherit from.